runtime/tests/shared/CompositeCacheSizesTests.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2001, 2017 IBM Corp. and others
+ * Copyright (c) 2001, 2020 IBM Corp. and others
```

Comment:
Nothing is changed in this file. So no need to update the copyright date.

Reply:
Reverted.

Reply:
This change is still in this PR. Seems that you didn't push your local change to your personal fork ?

Reply:
Wanted to push one more commit right away, but then decided to rework it.
I was reorganizing commits to make the history prettier. But replies were already posted.
Sorry for the confusion.